UsersApp/Models/PostalColde.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+
+enum PostalCode: Codable, CustomStringConvertible {
+    case int(Int)
+    case string(String)
```

Comment:
why do you need a `CustomStringConvertible ` for postal code?
This seems a bit advanced. Have you understood it?

Reply:
I had errors when fetching the postal code, because in the json file it is either an int or a string and I couldn't convert it directly to a string. I had understood it, although I have to use these protocols a few more times to master them